### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sf-jeff-lim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,111 +1,170 @@
...
+### Requirements
 
-## SDK Prerequisites
 * iOS 9.0 or later
 * Swift 4 or later, Objective-C
 * Xcode 9 or later, macOS Sierra or later.
 * Installation of **[Git Large File Storage](https://git-lfs.github.com)**
 
-> **IMPORTANT**: **MAKE SURE to install Git LFS before installing the pod**. Check the size of `WebRTC.framework` in SendBirdWebRTC folder. It MUST be over 800 MB. If the loaded SendBirdWebRTC framework is smaller than that, check the *Git Large File Storage* settings and download again. Refer to [the troubleshooting section](#Library-not-loaded-WebRTC.framework).
+> **IMPORTANT**: **MAKE sure to install Git LFS before installing the pod**. The size of `WebRTC.framework` in **SendBirdWebRTC** folder must be over 800 MB. If the size of the loaded `SendbirdWebRTC` framework is smaller than 800 MB, check the **Git Large File Storage** settings and download again. For further details, refer to SDK’s [troubleshooting section](#library-not-loaded-webrtcframework).
```
Comment (mininny) [inline README.md:45]: The B in `SendbirdWebRTC` should be capitalized(`SendBirdWebRTC`) as it's a framework name.

File: README.md [hunk 2]
```diff
@@ -1,111 +1,170 @@
...
 
-## Installation
-To use SendBird Calls, first add our custom-built WebRTC framework to the project. [Git Large File Storage](https://git-lfs.github.com) must be installed to use the WebRTC framework along with the SendBirdCalls framework.
 - Run `brew install git-lfs` in the project directory. 
 
 ### CocoaPods
-[CocoaPods](https://cocoapods.org/) is a dependency manager for Cocoa projects. For usage and installation CocoaPods instructions, visit [their website](https://cocoapods.org/). SendBirdCalls can be integrated into the Xcode project using CocoaPods by doing the following:
-1. Install CocoaPods into the project by running `pod init` in the project directory.
+
+[CocoaPods](https://cocoapods.org/) is a dependency manager for Cocoa projects. The `SendbirdCalls` framework can be integrated into the Xcode project using `CocoaPods` by doing the following.
```
Comment (mininny) [inline README.md:65]: `SendbirdCalls` should have the `B` capitalized as well.

Comment (mininny) [reply README.md:65]: I see few other `SendbirdCalls` without `B` capitalization; can you look at them as well? 🙏

**Review by mininny — COMMENTED** [review]
Thank you! I added few comments, and fixed few technical errors that I spotted in a [separate commit here](https://github.com/sendbird/sendbird-calls-ios/pull/2/commits/b1246459ad242da9e59bb8878b768434b2ed6197).

**Review by mininny — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -529,55 +640,58 @@ extension AppDelegate: DirectCallDelegate {
...
 
 ### iOS SDK Sizes
+
 | File       | Raw Files | Compiled Size |
 |------------|-----------|---------------|
 | Calls SDK  | 59.9 MB   | 6.18 MB       |
 | WebRTC SDK | 1.05 GB   | 11.2 MB       |
 
-*Last Updated: Sep 3rd, 2020*
+*- Last Updated: Sep 3rd, 2020*
```
Comment (jaesung-0o0) [inline README.md:649]: There's my mistake. Last updated must be modified to `Nov 10th, 2020`

Comment (jaesung-0o0) [inline README.md:649]:
```suggestion
*- Last Updated: Nov 10th, 2020*
```

File: README.md [hunk 4]
```diff
@@ -1,111 +1,170 @@
...
+
+## Configuring the application for the SDK
 
 ### Background Mode
-To support background operation, VoIP-enabled apps must have `Background Mode` enabled in the `Xcode Project > Signing&Capabilities` pane. Select the checkbox for `Voice over IP`.
 
-To receive push notifications, the app also must have `Push Notifications` enabled in the `Xcode Project > Signing&Capabilities` pane. 
+To support background operation, VoIP-enabled apps must have `Background Mode` enabled in the **Xcode Project** > **Signing & Capabilities** pane. Select the checkbox for **Voice over IP**.
+
+To receive push notifications, the app also must have **Push Notifications** enabled in the **Xcode Project **> **Signing & Capabilities** pane.
```
[the remark this answers is not in the document]
Comment (sf-jeff-lim) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (sf-jeff-lim) [reply README.md]: Done, all are checked!

File: README.md (hunk 3 above)
Comment (sf-jeff-lim) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (sf-jeff-lim) [reply README.md]: Done!

**Review by ipagong — APPROVED** [review]

**Review by jaesung-0o0 — APPROVED** [review]

**Pull request merged**